scheduler/pkg/store/pipeline/store.go

```diff
@@ -342,20 +342,19 @@ func (ps *PipelineStore) terminateOldUnterminatedPipelinesIfNeeded(pipeline *Pip
 func (ps *PipelineStore) SetPipelineState(name string, versionNumber uint32, uid string, status PipelineStatus, reason string, source string) error {
 	logger := ps.logger.WithField("func", "SetPipelineState")
 	logger.Debugf("Attempt to set state on pipeline %s:%d status:%s", name, versionNumber, status.String())
-	evts, err := ps.setPipelineStateImpl(name, versionNumber, uid, status, reason)
+	evts, err := ps.setPipelineStateImpl(name, versionNumber, uid, status, reason, source)
```

Comment:
I'm a bit confused, this line adds `source` which is not used in `setPipelineStateImpl`? Would passing this arg alone signal to the chainer that this needs to be processed?